### PR TITLE
add copyright notice to preresolved images

### PR DESCRIPTION
**Pull request by andygnewman**
cc: @matthew-andrews

### CODE REVIEW

File: server/lib/preresolve-images.js [hunk 1]
```diff
@@ -42,7 +42,9 @@ function templateMediaResource(mediaResource) {
 	const longdesc = mediaResource.longdesc ? ` alt="${mediaResource.longdesc}"` : '';
 	const width = mediaResource.pixelWidth ? ` width="${mediaResource.pixelWidth}"` : '';
 	const height = mediaResource.pixelHeight ? ` height="${mediaResource.pixelHeight}"` : '';
-	return `<img src="${mediaResource.binaryUrl}"${alt}${longdesc}${width}${height} />`;
+	const copyright = mediaResource.copyright && mediaResource.copyright.notice
+		? ` copyright="${mediaResource.copyright.notice}"` : '';
```
Comment (matthew-andrews) [inline server/lib/preresolve-images.js]: whilst the others are valid HTML attributes for images, i don't think that `copyright` is … perhaps `data-copyright-notice` or just `data-copyright`?

**Comment by matthew-andrews** [comment]
one small comment, otherwise :+1:

**Comment by andygnewman** [comment]
Learnt something new there about html attributes - you can only make them up if you put data- before them. Thanks

**Comment by matthew-andrews** [comment]
haha …  umm … kind of.

You can also have custom elements …

http://www.html5rocks.com/en/tutorials/webcomponents/customelements/

Then pretty much anything goes but there has to be a `-` in the tag name …

**Comment by matthew-andrews** [comment]
http://www.w3.org/TR/2011/WD-html5-20110525/elements.html#embedding-custom-non-visible-data-with-the-data-attributes

**Comment by matthew-andrews** [comment]
nice that we could merge this, by the time we come back from after christmas we'll have lots of lovely copyright notices coming through for our articles :-)

**Comment by andygnewman** [comment]
Also got some cleaning up to do post primaryBrand, which should now have worked through the system.

**Pull request merged**